components/server/src/workspace/workspace-starter.ts

```diff
@@ -414,6 +424,11 @@ export class WorkspaceStarter {
                 forceRebuild,
             );
         } catch (e) {
+            let failedReason: FailedInstanceStartReason = "other";
```

Comment:
@geropl this is the value for the alerts `rate(gitpod_server_instance_starts_failed_total{}[$__rate_interval])`.
Not sure this makes sense to have `other` if that's the default?

Reply:
You're :100: right, it doesn't (with the current quality of errors we're receiving). PR to fix this: https://github.com/gitpod-io/gitpod/pull/12574